frontend/app/routes/$lang+/_protected+/personal-information+/phone-number+/edit.tsx

```diff
@@ -77,6 +81,20 @@ export async function action({ context: { session }, params, request }: ActionFu
       .string()
       .min(1, { message: 'empty-phone-number' })
       .refine((val) => isValidPhoneNumber(val, 'CA'), { message: 'invalid-phone-format' }),
+
+    alternatePhoneNumber: z
+      .string()
+      .refine(
+        (val) => {
+          if (val.length === 0) {
+            return true;
+          } else {
+            return isValidPhoneNumber(val, 'CA');
+          }
+        },
+        { message: 'invalid-phone-format' },
+      )
+      .optional(),
   });
 
   const formData = Object.fromEntries(await request.formData());
```

Comment:
Can we change the use of `Object.fromEntries(..)` in favour of using `formData = await request.formData();` to be consistent with the rest of the application's `action(..)`?